contracts/core/EntryPoint.sol

```diff
@@ -285,23 +301,25 @@ contract EntryPoint is IEntryPoint, StakeManager {
     internal returns (uint256 gasUsedByValidateAccountPrepayment, address actualAggregator, uint256 deadline) {
     unchecked {
         uint256 preGas = gasleft();
-        MemoryUserOp memory mUserOp = opInfo.mUserOp;
-        _createSenderIfNeeded(opIndex, mUserOp, op.initCode);
+        _createSenderIfNeeded(opIndex, opInfo, op.initCode);
         if (aggregator == SIMULATE_FIND_AGGREGATOR) {
-            try IAggregatedAccount(mUserOp.sender).getAggregator() returns (address userOpAggregator) {
-                aggregator = actualAggregator = userOpAggregator;
+            //simulation marker after account creation,
+            // before calling account-specific methods (getAggregator, validateUserOp)
+            numberMarker();
```

Comment:
You added a 2nd marker to the code and the test below, but it's also mentioned in the EIP "In order to distinguish between them, there is a call to the NUMBER opcode (block.number), used as a delimiter between the validation functions." so I guess it must be also added there.

Reply:
Yes, I added a 2nd marker: this is the marker after creation, and the 2nd marker is after validateUserOp.
I don't really understand the comment.

Reply:
The comment was about having it in the code but not in the EIP.  Your commit today ([4714176](https://github.com/eth-infinitism/account-abstraction/pull/144/commits/47141764a634a09fd893bafc312386c1586d17d4)) fixed that.